ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_fft.py

```diff
@@ -0,0 +1,34 @@
+# local
+import ivy_tests.test_ivy.helpers as helpers
+from ivy_tests.test_ivy.helpers import handle_frontend_test
+
+
+# fft
+@handle_frontend_test(
+    fn_tree="jax.numpy.fft.fft",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("float"),

```

Comment:
This should be `get_dtypes("valid")`. If you get errors for any dtype then you will need to add the `@with_unsupported_dtypes` decorator to the function as described here https://lets-unify.ai/ivy/overview/deep_dive/data_types.html#supported-and-unsupported-data-types